arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/OpCodes.java

```diff
@@ -32,27 +33,29 @@
 public class OpCodes {
   private static final JsonConverter YAML_CONVERTER = JsonConverter.builder().enableYaml().build();
 
-  private static Map<Integer, OpCodeData> valueToOpCodeDataMap;
-  private static Map<OpCode, OpCodeData> opCodeToOpCodeDataMap;
+  private static Map<Integer, OpCodeData> valueToOpCodeDataMap = new ConcurrentHashMap<>();
+  private static Map<OpCode, OpCodeData> opCodeToOpCodeDataMap = new ConcurrentHashMap<>();
 
   /** Loads all opcode metadata from src/main/resources/opcodes.yml. */
   @SneakyThrows(IOException.class)
   public static void load() {
-    JsonNode rootNode =
-        YAML_CONVERTER
-            .getObjectMapper()
-            .readTree(OpCodes.class.getClassLoader().getResourceAsStream("opcodes.yml"))
-            .get("opcodes");
+    if (valueToOpCodeDataMap.isEmpty() || opCodeToOpCodeDataMap.isEmpty()) {
```

Comment:
why not doing it in a static initialization block as for the GasCalculator?

Reply:
It is possible but I see OpCodes more like a utility class here as we can call other other static methods. If we change it to a POJO, we would need to instantiate each time we need to get OpCodeData from an opcode